server/routes/projects.js

```diff
@@ -115,6 +115,21 @@ router
 			sendErrResp(res, e);
 		}
 	});
+router.route('/:projectId/likes').post(authenticateToken, async (req, res) => {
+	const { user } = req;
+	const userId = req.user._id;
+	const { username } = req.user;
+	const { projectId } = req.params;
+	try {
+		isValidObjectId(userId);
+		isValidUsername(username);
+		isValidObjectId(projectId);
+		const likeProject = await projectsData.likeProject(user, projectId);
+		res.json({ likeProject });
```

Comment:
post request needs to have the CREATED (201) status. please check how it is being done for other routes